swap_meet/vendor.py

```diff
@@ -1,2 +1,233 @@
 class Vendor:
-    pass
\ No newline at end of file
+    """
+    A class to represent a vendor.
+
+    ...
+
+    Attributes
+    ----------
+    inventory : list
+        list of strings representing items in inventory
+
+    Methods
+    -------
+    add(item):
+        Adds an item to the inventory list.
+        Returns item.
+    
+    remove(item):
+        Removes an item from the inventory list.
+        Returns item.
+        If item not in list, returns False.
+
+    get_by_category(category):
+        Creates list of items whose category attribute matches the input category.
+        Returns list. 
+
+    swap_items(vendor_to_swap_with, item_vendor_swaps, item_vendor_gets):
+        Removes item_vendor_swaps from vendor.inventory list and appends item_vendor_gets to list.
+        Removes item_vendor_gets from vendor_to_swap_with.inventory list and appends.item_vendor_swaps to list
+        Returns vendor's list.
+
+    swap_first_item(vendor):
+        swaps first items from my list and param vendor's list 
+        Returns true if both lists are not empty
+
+    return_best_by_category(category):
+        returns item with best condition in vendor list given the category
+
+    swap_best_by_category(other, my_priority, their_priority):
+        checks for your best item given other's priority
+        checks for their best item given your priority
+        swaps those items
+        returns true if swap occurs
+    
+    get_newest():
+        returns item in self.inventory with the lowest age attribute value
+
+    swap_by_newest(other):
+        swaps your's and other vendor's newest items
+        returns true if swap is successful
+    """
```

Comment:
In the future, feel free to keep these docstrings shorter (especially since you repeat the same descriptions in each method definition.). We primarily encourage as notes for yourself and to me if you have any.